src/caret_analyze/architecture/architecture.py

```diff
@@ -47,10 +46,11 @@ def __init__(
             file_type, file_path)
         loaded = ArchitectureLoaded(reader, ignore_topics)
 
-        self._nodes: Tuple[NodeStruct, ...] = loaded.nodes
-        self._communications: Tuple[CommunicationStruct, ...] = loaded.communications
-        self._executors: Tuple[ExecutorStruct, ...] = loaded.executors
-        self._path_manager = NamedPathManager(tuple(v.to_value() for v in loaded.paths))
+        self._nodes: List[NodeStruct] = loaded.nodes
+        self._communications: List[CommunicationStruct] = loaded.communications
+        self._executors: List[ExecutorStruct] = loaded.executors
+        assert isinstance(self._nodes, List)
```

Comment:
Let me confirm.
What is the intent of this Assert?
It seems to me that there is no problem without this Assert.
```suggestion
```

Reply:
@hsgwa 
It was described for confirmation of implementation.
It will be deleted later.

Reply:
@hsgwa 
Fix it in [06e9bb0](https://github.com/tier4/CARET_analyze/pull/156/commits/06e9bb075e64d397acb43697f6fc30c196c6868c)